.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/DateParser.cs

```diff
@@ -689,5 +691,10 @@ private int ConvertChineseYearToInteger(string yearChsStr)
 
             return year < 10 ? -1 : year;
         }
+
+        public List<DateTimeParseResult> FilterResults(string query, List<DateTimeParseResult> candidateResults)
+        {
+            return candidateResults;

```

Comment:
What's that function used?

Reply:
Only the basemeged parser really needs it, but in this version it needs to exist in the interface,
This is a naïve fix.